### ISSUE TITLE
Customized Tooltip Is Not Defined and Is Not Announced by Screen Reader: A11y_ Visual Studio Code Services_Open Source AI code editor screen _Usable.

### ISSUE THREAD

**Issue by AnKushSingh05** [Microsoft/vscode-docs#9311]
### GitHub Tags: 

#A11yTCS; #A11yUsable; #DesktopWeb; #ChromiumEdge; #Win11; #Visual Studio Code Services; #E2E_Visual Studio Code Services_Web_Jan26; #Closed; #Regressed:03-11-26; #A11yFixed; 

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."  

### Environment Details: 

Application Name: Visual Studio Code Services 
URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0)  
Chromium Edge: Version 144.0.3719.104 (Official build) dev (64-bit) 
Edition: Windows 11 Enterprise Insider Preview 
Version: 24H2 
OS build: 26558.1000 
Narrator, NVDA 2025.3.2(2025.3.2.53588). 

### Repro steps: 

1. Turn on Screen reader and Hit the URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0) 
2. Tab navigate to WEB, License, and Privacy Statement links. 
3. Observe that Customized Tooltip Is Not Defined and Is Not Announced by Screen Reader. 

### Actual Result: 

When hovering or focusing on the control, no customized tooltip is defined. As a result, the screen reader does not announce any tooltip information, leaving users unaware of the control’s purpose or additional context. 
**NOTE: Issue is NO repro with NVDA.**

### Expected Result: 

A properly defined and accessible customized tooltip should be provided, and the screen reader should announce the tooltip text whenever the user hovers or moves keyboard focus to the control. 

### User Impact: 

Users who rely on screen readers will not receive any tooltip information, leaving them unaware of the control’s purpose or additional context. This can lead to confusion, incorrect actions, and reduced task efficiency—especially for users with visual impairments who depend on announced tooltips for guidance. 

### Attachment:

https://github.com/user-attachments/assets/f3926ca4-a667-4458-8721-0a224c70d506

<img width="960" height="540" alt="Image" src="https://github.com/user-attachments/assets/ba75e067-cfee-4fc6-b7bf-3de8df53596a" />

**Comment by ntrogh** [comment]
@AnKushSingh05 Issue should be resolved

**Comment by AnKushSingh05** [comment]
Verified in below environment, Issue is Fixed. Hence closing the bug.
[https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0)

https://github.com/user-attachments/assets/2338484a-039d-4e4d-bf1f-daa5b3236635
